index.js

```diff
@@ -0,0 +1,213 @@
+var exerciseString = "This is a dummy string for testing";
+var exerciseArray = exerciseString.split(" ");
+
+console.log("Mini-assignment: week 12 day 02");
+console.log("===============================\n");
+console.log(`String to use: ${exerciseString}`);
+console.log(`Array to use: ${exerciseArray}`);
+
+console.log(`\nPOP function
+===========`);
+console.log(`Last element: ${arrayPop(exerciseArray)}`);
+console.log(`New array: ${exerciseArray}`);
+
+function arrayPop(array) {
+  let last = array[array.length - 1];
+  let temporal = [];
```

Comment:
you could use ````array.length = array.length - 1```` to eliminate the last commit.

Reply:
Yes, I've seen Valeria and Ricardo solution, they used `array.length--`